x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/wrap_suppressed_alerts.ts

```diff
@@ -59,12 +60,12 @@ export const wrapSuppressedAlerts = ({
       fields: event.fields,
     });
 
-    const id = objectHash([
+    const id = generateId(
       event._index,
       event._id,
-      `${spaceId}:${completeRule.alertId}`,
-      suppressionTerms,
-    ]);
+      String(event._version),
```

Comment:
Why the addition of `event._version` here?

Reply:
it's to match generated id in https://github.com/elastic/kibana/blob/8.14/x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/factories/wrap_hits_factory.ts#L46-L51,
so alerts can be deduplicated